zipkin4net.sln

```diff
@@ -0,0 +1,81 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Since every project generates an independant nuget, is it a good idea to have a solution containing every project ?